hr_employee_birth_name/views/hr_employee_view.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version = "1.0" encoding="utf-8"?>
+<!-- Copyright 2017 Denis Leemann
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo>
+
+    <record id="view_employee_form_inherit_extension" model="ir.ui.view">
+      <field name="name">view.employee.form.inherit.extension</field>
+      <field name="model">hr.employee</field>
+      <field name="inherit_id" ref="hr.view_employee_form"/>
+      <field name="arch" type="xml">
+
+        <xpath expr="//field[@name='marital']" position="after">
```

Comment:
Direct use `field`

Reply:
Excuse me, I do not understand this change, you prefer that I remove the xpath?

Regards

Reply:
Yes remove `xpath` and use `<field name="marital" position="after">`